backend/user/views.py

```diff
@@ -150,6 +156,70 @@ def delete(self, request, admin_id):
         return Response(status=status.HTTP_204_NO_CONTENT)
 
 
+class SaveUnitApiView(APIView):
+    """
+    An endpoint to handle saving a vehicle to a bidder's saved vehicle list,
+    as well as retrieving a list of all saved vehicles
+    """
+
+    def get(self, request, **kwargs):
+        # Replace this line to obtain user id once authentication has been implemented
+        bidder_id = kwargs.get("bidder_id")
+        bidder = get_object_or_404(Bidder, id=bidder_id)
+
+        saved_units = SavedUnits.objects.filter(bidder_id=bidder)
+        vehicle_list = [
+            saved_unit.content_object
+            for saved_unit in saved_units
+            if isinstance(saved_unit.content_object, Vehicle)
+        ]
+
+        vehicle_data = [{"id": vehicle.id} for vehicle in vehicle_list]
+
+        return Response({"vehicles": vehicle_data}, status=status.HTTP_200_OK)
```

Comment:
@kevinrczhang Since this is fetching all vehicles that a bidder has saved. This should go in its own View and have its own endpoint `<uuid:auction_id>/bidders/<uuid:bidder_id>/vehicles`